client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -157,6 +161,18 @@ def _set_scene_path(self, current_file, use_published):
         self.scene_path = file_path
         self.log.info("Using {} for render/export.".format(file_path))
 
+    def _set_job_output(self, instance, job_info):
```

Comment:
I wonder if this is better labeled `add` or `append` to be clear that it does not override the list completely.
```suggestion
    def _append_job_output(self, instance, job_info):
```

Reply:
ok